docs/cow-protocol/tutorials/widget.md

```diff
@@ -59,13 +59,21 @@ import {cowSwapWidget, CowSwapWidgetParams} from '@cowprotocol/widget-lib'
 const widgetContainer = document.getElementById('cowswap-widget')
 
 const params: CowSwapWidgetParams = {
-  partnerFeeBips: '50' // 0.5%
+  partnerFee: {
+      bps: 50, // 0.5%
+      recipient: '0x0000000000000000000000000000000000000000' // Fee destination address
+  }
 }
 
 cowSwapWidget(widgetContainer, params)
 ```
 
-> **Coming soon, please get in touch to sign up to the beta program [here](https://calendly.com/crystal-cow/cow-swap-widget)**
+The fee in basis points (BPS). One basis point is equivalent to 0.01% (1/100th of a percent).  
+The recipient is the address to which the fee will be sent.
```

Comment:
Can we mention it's capped at 1% (100 bps)?

Reply:
Nice catch! Added